### ISSUE TITLE
"Go to definition" does not work if caret is at the end of a qualified symbol, right before dot

### ISSUE THREAD

**Issue by vasily-kirichenko** [Microsoft/visualfsharp#1858]
(current master)

If caret (`|` here) is at the end of a symbol, Go to definition does not work

```fsharp
module Module1 =
    let foo x = x

let _ = Module1|.foo 1
```

![image](https://cloud.githubusercontent.com/assets/873919/20646859/813ff3a0-b495-11e6-8392-999a1fbf753c.png)

It does work if caret is at start or in the middle:

```fsharp
let _ = |Module1.foo 1
let _ = Mod|ule1.foo 1
```

**Comment by dsyme** [comment]
Odd, I thought that was fixed.  I'll take a look

**Comment by dsyme** [comment]
Ok, it was only for a long identifier ``Module1.foo`` name.  Fixed in #1855
